models/NextAuthUser.js

```diff
@@ -0,0 +1,44 @@
+// This file overrides the custom defined User Schema Next Auth puts into our database when a user signs in with SSO
```

Comment:
Can we discuss more about how this works in tomorrow's meeting before merging? Not sure how this works behind the scenes.

Reply:
I found this https://next-auth.js.org/tutorials/typeorm-custom-models but I'm not sure why you're setting other weird like username, phoneNumber, etc above the constructor.

Reply:
Gorgeous